src/main/java/io/confluent/connect/jdbc/JdbcSourceConnectorConfig.java

```diff
@@ -161,6 +167,8 @@ public static ConfigDef baseConfigDef() {
                 Importance.HIGH, MODE_DOC, MODE_GROUP, 1, Width.MEDIUM, MODE_DISPLAY, Arrays.asList(INCREMENTING_COLUMN_NAME_CONFIG, TIMESTAMP_COLUMN_NAME_CONFIG, VALIDATE_NON_NULL_CONFIG))
         .define(INCREMENTING_COLUMN_NAME_CONFIG, Type.STRING, INCREMENTING_COLUMN_NAME_DEFAULT, Importance.MEDIUM, INCREMENTING_COLUMN_NAME_DOC, MODE_GROUP, 2, Width.MEDIUM, INCREMENTING_COLUMN_NAME_DISPLAY,
                 MODE_DEPENDENTS_RECOMMENDER)
+        .define(INCREMENTING_COLUMN_USE_PRIMARY_KEY_CONFIG, Type.BOOLEAN, INCREMENTING_COLUMN_USE_PRIMARY_KEY_DEFAULT, Importance.MEDIUM, INCREMENTING_COLUMN_USE_PRIMARY_KEY_DOC, MODE_GROUP, 2, Width.MEDIUM, INCREMENTING_COLUMN_USE_PRIMARY_KEY_DISPLAY,
```

Comment:
We should change 2 to 3. The number represents the order of the config in the group.